tests/generic_config_updater/files/patch_sorter_test_failure.json

```diff
@@ -0,0 +1,71 @@
+{
+    "EMPTYING_A_CONFIGDB_TABLE__FAILURE": {
+        "desc": "Emptying a configdb table fails because empty tables are not allowed in configdb.",
```

Comment:
>allowed [](http://example.com/codeflow?start=78&length=7)

I understand this is not allowed. Do you expect user to remove `/ACL_TABLE/`? If yes, add it into `desc` ?

Reply:
This test is not about emptying `ACL_TABLE` it is about emptying any configdb table. I think adding `ACL_TABLE` to the name will be confusing.

Reply:
User can delete any configdb table, as long as it passes YANG validation.

Reply:
Offline discussion:
[15:00] Mohamed Ghoneim
Ah, you mean what should the user do in case they encounter this error? good point

[15:00] Qi Luo
yes

Updated